lib/Froxlor/Cron/Http/DomainSSL.php

```diff
@@ -105,7 +105,9 @@ public function setDomainSSLFilesArray(array &$domain = null)
 					$_fh = fopen($filename, 'w');
 					fwrite($_fh, $dom_certs[$type]);
 					fclose($_fh);
-					chmod($filename, 0600);
```

Comment:
depending on the umask/defaults, it might be a good idea to also explicitly set the 0644 for any other type

Reply:
OK, maybe it is better to have a defined state than use the default.